pyproject.toml

```diff
@@ -9,10 +9,10 @@ dependencies = [
     "fastapi>=0.115.8",
     "geojson-pydantic>=1.2.0",
     "pydantic>=2.10.4",
+    "rustac",
```

Comment:
do you expect breaking change?

I think it's fine to it open for now but if we do a pypi release we may want to change this/

Reply:
We will harden the dependency once we're not mucking **rustac** around as much ... for now we'll just go to an unreleased SHA.